source/Octopus.Client/Repositories/BasicRepository.cs

```diff
@@ -47,7 +49,8 @@ public void Delete(TResource resource)
 
         public void Paginate(Func<ResourceCollection<TResource>, bool> getNextPage, string path = null, object pathParameters = null)
         {
-            client.Paginate(path ?? client.RootDocument.Link(CollectionLinkName), pathParameters ?? new { }, getNextPage);
+            var parameters = ParameterHelper.CombineParameters(LimitedToSpacesParameters, pathParameters);
+            client.Paginate(path ?? client.RootDocument.Link(CollectionLinkName), parameters, getNextPage);
```

Comment:
What about all of the other `GET` methods that require these parameters? eg `GetAll`, and any methods in derived repositories?